components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/resources/web/entitlement/add-policy.jsp

```diff
@@ -33,7 +34,7 @@
             String message = resourceBundle.getString("policy.editor.config.update");
             %>
             <script type="text/javascript">
-                CARBON.showInfoDialog('<%=message%>', null, null);
+                CARBON.showInfoDialog('<%=Encode.forJavaScriptAttribute(message)%>', null, null);
```

Comment:
This message is directly obtained from the resource bundle. Thus, not necessary to encode. If encoding we have to first encode for html and then encode for javascript block because the passed parameter is directly injected to an html